src/main/haskell/kore/src/Kore/MetaML/Lift.hs

```diff
@@ -435,6 +435,41 @@ liftMetaSentence (SentenceAxiomSentence as) =
             UnifiedMetaPattern _   -> liftedPattern
             UnifiedObjectPattern _ ->
                 Fix (apply (provableHead axiomSort) [liftedPattern])
+liftMetaSentence (SentenceClaimSentence as) =
```

Comment:
I wonder whether here you could have extracted some code for `SentenceAxiom` and used it for both `SentenceAxiomSentence` and `SentenceClaimSentence`